live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/transform/response/ATSTransformResponseHandler.java

```diff
@@ -59,41 +50,34 @@ public class ATSTransformResponseHandler extends RouteBuilder
     private static final String ROUTE_ID = "transform-events-consumer";
 
     private final IngestContentCommandHandler ingestContentCommandHandler;
-    private final IngestNodeCommandHandler ingestNodeCommandHandler;
     private final IntegrationProperties integrationProperties;
 
     @Override
     public void configure()
     {
+        // @formatter:off
         SecurityContext securityContext = SecurityContextHolder.getContext();
         from(integrationProperties.alfresco().transform().response().endpoint())
                 .routeId(ROUTE_ID)
                 .log(DEBUG, "Received transform completed event : ${body}")
                 .unmarshal()
                 .json(JsonLibrary.Jackson, TransformResponse.class)
                 .process(exchange -> SecurityContextHolder.setContext(securityContext))
-                .process(this::uploadContentRendition)
-                .process(this::updateContentLocation)
```

Comment:
I had to replace it with single process() because previously it was unit-untestable